regression/Makefile

```diff
@@ -2,22 +2,16 @@
 # running tests in parallel, it is important that these directories are
 # listed with decreasing runtimes (i.e. longest running at the top)
 DIRS = cbmc \
-       cbmc-java \
        goto-analyzer \
        ansi-c \
-       jbmc-strings \
        goto-instrument \
        cpp \
-       strings-smoke-tests \
        cbmc-cover \
        goto-instrument-typedef \
        smt2_solver \
        strings \
        invariants \
-       goto-diff \
```

Comment:
This seems wrong. If there are Java-only bits then there should be a goto-diff-java directory instead.

Reply:
It looks like it's 11 out of 17 tests in goto-diff that use java, but I agree, it seems wrong to be disabling non-java tests in this PR! I've replicated this particular change in the CMakeFile fix up commit, but that doesn't mean I agree with that change :-)